tutorials/python/python-video-generation/code/Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+flake8 = "*"
+pylint = "*"
+autopep8 = "*"
+
+[packages]
+moviepy = "*"
+gizeh = "*"
```

Comment:
@Larox Could you please specify the version that you're using here?
```Pipfile
moviepy = "~=1.0.0"
gizeh = "~=0.1.11"
```

Just to be clear with the version of the dependencies used. Sounds good?

Reply:
Sure thing!